docs/modules/ROOT/pages/reference/extensions/sql.adoc

```diff
@@ -40,6 +40,14 @@ Or add the coordinates to your existing project:
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
 
+== Camel Quarkus limitations
+
+While you can use any of the available database types in JVM mode, Oracle database type negatively affects serialization in native mode.
+See quarkus issue https://github.com/quarkusio/quarkus/issues/20396[#https://github.com/quarkusio/quarkus/issues/20396] for more dtails.
+
+https://camel.apache.org/components/next/sql-component.html#_using_the_jdbc_based_aggregation_repository[Aggregation repository] does not work in native mode with Oracle db, because of this issue.
```

Comment:
I thought it was possible to link this in a relative way but can't find it anymore.

Reply:
Relative link would be much better. I quickly went through doc files in extensions and didn't find it either.

Reply:
Tentative: maybe @jamesnetherton or @zhfeng would remember having done that in a previous PR ?

Reply:
`xref:{cq-camel-components}::sql-component.adoc` as proposed below

Reply:
done